tasks/server.yml

```diff
@@ -24,6 +24,17 @@
 - name: check if log file exists
   stat: path={{ redis_logfile }}
   register: logfile_stat
+  
+- name: Redis | Make sure pidfile directory exists and have correct owner
+  sudo: yes
+  sudo_user: 'root'
```

Comment:
Remove these sudo options from the tasks; they should be set at the playbook level.